tests/unit/python/fledge/common/test_configuration_manager.py

```diff
@@ -1198,8 +1328,15 @@ async def async_mock(return_value):
         category_name = 'catname'
         item_name = 'itemname'
         new_value_entry = 'newvalentry'
+        
+        # Changed in version 3.8: patch() now returns an AsyncMock if the target is an async function.
+        if sys.version_info.major == 3 and sys.version_info.minor >= 8:
+            _rv = await async_mock({'value': 'test', 'description': 'Test desc', 'type': 'boolean', 'default': 'test'})
```

Comment:
Better to add `_payload = {'value': 'test', 'description': 'Test desc', 'type': 'boolean', 'default': 'test'}`
and reuse further